### PR TITLE
fix: Portable Experiences UI issues

### LINKED ISSUE #2143: Portable experiences UI issues
- When turning on and off the toggle doesn't change but the PX dissapears correctly. It is confusing to see both the Eye and the toggle
- When resizing, the canvas of the PX seems to not be reacting to the new screen size

**Slack thread:** https://decentralandteam.slack.com/archives/C01QVR7TJK1/p1649255258091509

### PR DESCRIPTION
**Pull request by sandrade-dcl**
## What does this PR change?
Fixes #2143 

When turning on and off the toggle didn't change but the PX dissapeared correctly. Now the status of the UI is synchronized with its PEX in execution.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/portable-experiences-ui-issues
2. Equip a smart wearable.
3. Notice the PEX is loaded in the Experiences viewer.
4. Activate/Deactivate the the toggle in the PEX.
5. Notice the PEX is being enabled/disabled and the UI's behaviour is working correctly.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Review by davidejensen — APPROVED** [review]
Code LGTM and all works fine!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExperiencesViewer/Scripts/ExperiencesViewerComponentController.cs [hunk 1]
```diff
@@ -237,6 +237,12 @@ internal void OnUserProfileUpdated(UserProfile userProfile)
                 lastDisablePEXSentToKernel = pausedPEXScenesIds;
                 WebInterface.SetDisabledPortableExperiences(pausedPEXScenesIds.ToArray());
             }
+
+            List<ExperienceRowComponentView> loadedExperiences = view.GetAllAvailableExperiences();
+            foreach (ExperienceRowComponentView experience in loadedExperiences)
```
Comment (AjimenezDCL) [inline unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExperiencesViewer/Scripts/ExperiencesViewerComponentController.cs:242]: Nitpicking here but let's use a `for`.

**Review by AjimenezDCL — APPROVED** [review]
Left a small request, but looking good!

File: unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExperiencesViewer/Scripts/ExperiencesViewerComponentController.cs (hunk 1 above)
Comment (sandrade-dcl) [reply unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExperiencesViewer/Scripts/ExperiencesViewerComponentController.cs]: Done in https://github.com/decentraland/unity-renderer/pull/2174/commits/a61743addd844e68fb665f47f81738851ec580dd. Thanks!

**Pull request merged**